### ISSUE TITLE
CIFAR-10 dataset images

### ISSUE THREAD

**Issue by mehak126** [vineetm/ell-881-2018-deep-learning#45]
Please correct me if I'm wrong, but don't the dimensions of the input images to alexnet have to be 256X256? The CIFAR-10 images have a size 32X32. If we resize them to 256X256, will there be too much distortion?

**Comment by anshumitts** [comment]
That's why we have interpolation techniques

**Comment by raghavsi** [comment]
Yes I am also surprised that they upscale 32x32...but Vgg is 256x256
